packages/ketcher-core/src/application/editor/actions/rgroupAttachmentPoint.ts

```diff
@@ -0,0 +1,126 @@
+import assert from 'assert';
+import { ReStruct } from 'application/render';
+import { Struct } from 'domain/entities';
+import {
+  RGroupAttachmentPointAdd,
+  RGroupAttachmentPointRemove,
+} from '../operations';
+import { Action } from './action';
+import { fromAtomsAttrs } from '.';
+
+export function fromRGroupAttachmentPointUpdate(
+  restruct: ReStruct,
+  atomId: number,
+  attpnt,
+) {
+  const action = new Action();
+  action.mergeWith(fromRGroupAttachmentPointsDeletionByAtom(restruct, atomId));
+  action.mergeWith(fromRGroupAttachmentPointAddition(restruct, attpnt, atomId));
+  return action;
+}
+
+function fromRGroupAttachmentPointAddition(
+  restruct: ReStruct,
+  attpnt,
```

Comment:
Please use the full name for variables/parameters. Shorten ones can confuse newcommers in future

Reply:
`attpnt` is an attribute of atom, I'm not sure it was named after some specification or it was an abbreviation of `rgroupAttachmentPoint`, so in this situation, I think keeping the name `attpnt` can make developers understand it's the same as `attpnt` in entity atom. 
Something needs to do in the future: https://github.com/epam/ketcher/issues/2962